pkgs/tools/misc/chafa/.#default.nix

```diff
@@ -0,0 +1 @@
+mog@ford.4882:1553654883
```

Comment:
This looks like it snuck in by accident?

Reply:
yup cleaning that now